telebot/types.py

```diff
@@ -8050,7 +8137,7 @@ class ReactionTypeCustomEmoji(ReactionType):
     :rtype: :class:`ReactionTypeCustomEmoji`
     """
 
-    def __init__(self, custom_emoji: str) -> None:
+    def __init__(self, custom_emoji: str, **kwargs) -> None:
```

Comment:
btw, custom_emoji should be turned to custom_emoji_id:
https://core.telegram.org/bots/api#reactiontypecustomemoji

Reply:
Thanks, fixed.